src/main/telemetry/frsky.c

```diff
@@ -190,7 +191,14 @@ static void sendThrottleOrBatterySizeAsRpm(void)
 {
     sendDataHead(ID_RPM);
     if (ARMING_FLAG(ARMED)) {
-        serialize16(rcCommand[THROTTLE] / BLADE_NUMBER_DIVIDER);
+        if(rcCommand[THROTTLE] <= escAndServoConfig->minthrottle){
```

Comment:
This is only pointer, it points to random location until initialized.

You should use `masterConfig` from 'config/config_master.h'. You must include other '.h' to resolve structures included in master config. Just add header files with structures from error messages ..